exo-setup/src/docker-setup.ls

```diff
@@ -39,7 +39,7 @@ class DockerSetup extends EventEmitter
 
   _docker-file-exists: ~>
     try
-      fs.access-sync path.join(@config.root, 'Dockerfile')
+      fs.exists-sync path.join(@config.root, 'Dockerfile')
```

Comment:
From the docs: 
`Note that fs.exists() is deprecated, but fs.existsSync() is not. (The callback parameter to fs.exists() accepts parameters that are inconsistent with other Node.js callbacks. fs.existsSync() does not use a callback.)`

 Would you prefer we still go with `stat-sync`?